bin/pygrb/pycbc_pygrb_plot_injs_results

```diff
@@ -309,6 +309,7 @@ if opts.newsnr_threshold:
         found_inj[key] = found_inj[key][~rw_snr_cut]
     for ifo in ifos:
         found_inj[ifo+'/end_time'] = found_inj[ifo+'/end_time'][~rw_snr_cut]
+    found_inj['reweighted_snr'] = found_inj['reweighted_snr'][~rw_snr_cut]
```

Comment:
Do you not need to also modify `missed_inj`? I.e. could you simply not add `'reweighted_snr'` to the first for loop into this if block?

Reply:
We plot all missed injections ("genuinely" missed and vetoed) further down via `missed_inj`.  But injections that were never found do not have a reweighted SNR, so either I make two separate loops or I keep `'reweighted_snr'` separate.

Reply:
This is a one line change, it is more minimal than the other approach, but I am happy to proceed either way.

Reply:
Sure, looks ok like this.